index.html

```diff
@@ -14,6 +15,185 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container-home">
```

Comment:
You can have multiple class names, space separated, just like you did below with `box box1`, try that here with `container home`, the idea being that you can apply global styling to the container, and more specific styles to the `home` class. This becomes more important as you build multiple pages of a website.